docs/source/references/apollo-engine.md

```diff
@@ -0,0 +1,158 @@
+
+<h3 id="add-organization">Adding an organization</h3>
+
+If you’re looking for a GitHub organization that you’re a member of and don’t see it in Engine, it’s likely that Engine does not have read access for that organization.
+
+If you want to Add or Remove an organization from Engine, you should manage those settings on GitHub. There, you will be able to Grant or Revoke access to Engine for organizations you can administer. For organizations you do not administer, you can Request access to Engine and the administrators will receive a request by E-mail.
+
+<h3 id="github-permissions">GitHub permissions</h3>
+
+GitHub’s OAuth service is used for read-only information about organizations and users. Engine does not need access rights to your source code or to any other sensitive data in its login system.
+
+If your Engine account is owned by a GitHub organization, then Engine will allow all members of that organization to access the account. As you add or remove team members from your Github org, Engine will know about that and accordingly update the authorization for those users.
+
+<h3 id="other-logins">Support for non-GitHub login</h3>
+
+Engine was built to mirror GitHub’s login and does not currently support other forms of login or team creation. Our current recommendation for teams who want to use Engine but do not use GitHub is:
+
+1. Create a free GitHub organization (empty, public).
+2. Add your desired team members to the organization.
+3. Hide membership of the organization if desired.
+4. Use this organization to collaborate on Engine.
+
+
+<h2 id="services">Services</h2>
+
+A service in Engine is an entity that is provisioned in the Engine cloud service that your server can report performance metrics and schema versions to via an API key. The information reported by your server is processed and stored in the Engine cloud service and made accessible to you through the Engine interface.
+
+<h3 id="creating-services">Creating a service</h3>
+
+Services in Engine have globally unique IDs, so we recommend that you prefix your service ID with the name of your company or organization. To create a service, you will need to select an account for that service to belong to. All members of the account will be able to see the service’s data and settings options.
+
+You can transfer services between any Engine account that you’re a member of. To transfer a service, visit its Settings page and change the “Endpoint owner” to whichever account you’d like.
+
+<h3 id="environments">Managing environments</h3>
+
+Services in Engine represent different projects/applications. Environments within your application can be tracked using [schema tags] (//TODO(dman): find link and figure out this recommendation) and managed within a single service.
```

Comment:
Let's try to make the relationship between a service and its schema tags super clear because I think it's going to trip a lot of people up.